Poke/App.tsx

```diff
@@ -83,10 +88,14 @@ const App = (props: any) => {
       <SafeAreaView>
         <ScrollView contentInsetAdjustmentBehavior="automatic" style={styles.scrollView}>
           <View style={styles.body}>
-            <Text style={styles.simple}>Bem vindo(a) à Taqtile!</Text>
-            <EmailInput text={email} onTextChange={setEmail} />
-            <PasswordInput text={password} onTextChange={setPassword} />
-            <SubmitButton onTap={() => {handleSubmit(props)}} />
+            {isLoading ? <ActivityIndicator size={'large'} color="#000000"/> : 
```

Comment:
Quando a prop é uma string, podemos omitir as chaves `{}`:
```suggestion
            {isLoading ? <ActivityIndicator size='large' color='#000000' /> : 
```

É interessante também mantermos o padrão de aspas simples ou duplas em todas as strings. O lint não reclamou disso?

Reply:
Na verdade nao haha